addons/website_slides_survey/tests/test_course_certification_remove_membership.py

```diff
@@ -93,12 +97,12 @@ def test_course_certification_karma(self):
         self.slide_certification.with_user(self.user_portal)._action_mark_completed()
         self.assertEqual(10, self.user_portal.karma, 'User Karma should be updated to 10')
 
-        # Removing of membership should remove Karma Gained from the Course
+        # While removing of membership, Karma will be same from the Course
```

Comment:
These changes do not belong to this commit.

Reply:
@awa-odoo Should I squash all the commits into one commit with a proper commit message? So won't produce any uncertainty regarding the commit message or wrong commit for this change etc. WHat's say?

Reply:
@awa-odoo ok will keep that 3 commits with proper commit messages and changes. Yes, the fourth one is regarding karma points forgot to push the latest changes regarding karma that's why it is creating confusion but I will take care of it in the future. Thanks.